pyproject.toml

```diff
@@ -19,6 +19,7 @@ rdflib = "^5.0.0"
 importlib_metadata = {version = "^1.5.0", python = "<3.8"}
 concepttordf = "^1.0.0"
 rdflib-jsonld = "^0.5.0"
+nox-poetry = "^0.8.4"
```

Comment:
Kvifor trenger vi nox-poetry i pyproject.toml?

Mi erfaring er at bygge-tools kun skal installeres lokalt (på utviker-pc) og i workflow.
Og den er vel i alle tilfelle i dev-dependencies.

Reply:
Dette er litt mystisk. Jeg har egentlig bare kjørt disse kommandoene for å oppdatere nox-poetry. Jeg mistenker at det kommer derfra, men jeg kan prøve å gjøre det hele på nytt og se om det er en konsekvens av den oppdateringen..

Reply:
Jeg tror jeg bør decline denne PR'en. Jeg sjekket ut master nå uten disse endringene og nox bygger fint lokalt hos meg nå, som jo var målet med oppdateringene, så dette er ting som ikke er nødvendige men som pga noe jeg har gjort har sneket seg inn.